code/frameworks/angular/src/client/docs/compodoc.ts

```diff
@@ -146,11 +146,13 @@ export const extractType = (property: Property, defaultValue: any) => {
       return { name: compodocType };
     case undefined:
     case null:
-      return { name: 'void' };
+      return { name: 'other', value: 'void' };
     default: {
       const resolvedType = resolveTypealias(compodocType);
       const enumValues = extractEnumValues(resolvedType);
-      return enumValues ? { name: 'enum', value: enumValues } : { name: 'object' };
+      return enumValues
+        ? { name: 'enum', value: enumValues }
+        : { name: 'other', value: 'empty-enum' };
```

Comment:
does 'empty-enum' have some significance? some magic string something else uses?
If so, should it be a constant?

Reply:
My understanding is the `value` of an `'other'` SBType is there to dump essentially debugging info into. It's not used in practice AFAIK, but I could be wrong, cc @shilman